sdk/firstload/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "highlight.run",
-	"version": "7.3.8",
+	"version": "7.3.10",
```

Comment:
Skip 9? 😆

Reply:
hehe i knew about #5569 which was gonna take that one ;)